### PR TITLE
chore: add null-loader for tests

### PR DESCRIPTION
**Pull request by frassinier**
**Please check if the PR fulfills these requirements**
- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] The code coverage on new code is > 75 % for backend and > 95% for frontend
- [x] The new code does not introduce new technical issues (sonar / eslint)
- [x] Functional tests have been performed

**Please check the browsers you've tested on**
- [ ] Chrome, Firefox, Safari, Edge, IE11
- [ ] No, that's bad, this PR should not be merged !
- [x] No, and no need to (backend changes only)

**(Optional) What is the current behavior?**
(Additional information to the Jira)
`npm test  68,08s user 2,54s system 107% cpu 1:05,97 total`

**(Optional) What is the new behavior?**
(Additional information to the Jira)
`npm test  54,69s user 1,91s system 104% cpu 54,117 total`

**(Optional) Other information**:

### CODE REVIEW

**Comment by build-talend-tdp** [comment]
Pull request test result: [success](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/74/) [:star:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/74/)

File: dataprep-webapp/package.json [hunk 1]
```diff
@@ -105,6 +105,7 @@
     "ng-annotate-loader": "^0.2.0",
     "ngtemplate-loader": "^1.3.1",
     "node-sass": "^4.5.0",
+    "null-loader": "^0.1.1",
```
Comment (charlesnguyen) [inline dataprep-webapp/package.json]: why use 'null-loader'? we can use nothing if isTestMode is true?

Comment (frassinier) [inline dataprep-webapp/package.json]: nop, it will fails when it will try assets resolution :(

**Comment by build-talend-tdp** [comment]
Pull request test result: [failure](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/90/) [:x:](https://ci-tdp.datapwn.com/job/data-prep_pull-request_master/90/)

**Pull request merged**